gallery/workbook/Azure Monitor.json

```diff
@@ -105,8 +105,7 @@
 					"id": "Workbooks/Azure Stack HCI/Dell/MultiCluster",
 					"author": "Dell",
 					"description": "Multi Cluster Event Monitoring for Dell APEX Cloud Platform for Microsoft Azure",
-					"name": "Multi Cluster Event Monitoring for Dell APEX Cloud Platform for Microsoft Azure",
-					"isPreview": true
+					"name": "Multi Cluster Event Monitoring for Dell APEX Cloud Platform for Microsoft Azure"
```

Comment:
why is the name the same as the description?  the name should be something short so it isn't "multi cluster even....." and then the description has the same content?

Reply:
Thanks John, will review this with documentation